config/typo3.rules

```diff
@@ -1,46 +1,43 @@
 # Harden the `chmod` function
 sp.disable_function.function("chmod").param("mode").filename_r("typo3/sysext/core/Classes/Utility/GeneralUtility.php$").value_r("^[0-9]{2}6$").allow();
 sp.disable_function.function("chmod").param("mode").value_r("^[0-9]{2}[67]$").drop();
-sp.disable_function.function("chmod").param("mode").value_r("o\\+w$").drop();
-
-# Prevent various `mail`-related vulnerabilities
-sp.disable_function.function("mail").param("additional_parameters").value_r("\\-").drop();
 
 ##Prevent various `include`-related vulnerabilities
-sp.disable_function.function_r("^(?:require|include)_once$").value_r("\\.(?:php|php7|inc|tpl)$").allow();
-sp.disable_function.function_r("^require|include$").value_r("\\.(?:php|php7|inc|tpl)$").allow();
+sp.disable_function.function_r("^(?:require|include)_once$").value_r("\.php$").allow();
+sp.disable_function.function_r("^require|include$").value_r("\.php$").allow();
 sp.disable_function.function_r("^(?:require|include)_once$").drop();
 sp.disable_function.function_r("^require|include$").drop();
 
 # Prevent `system`-related injections
-sp.disable_function.function("system").param("command").value_r("[$|;&`\\n]").drop();
-sp.disable_function.function("shell_exec").param("command").filename_r("vendor/nikic/php-parser/grammar/rebuildParsers.php").value_r("2>&1").allow();
+sp.disable_function.function("system").drop();
+sp.disable_function.function("shell_exec").param("command").filename_r("vendor/nikic/php-parser/grammar/rebuildParsers.php$").value_r("2>&1").allow();
```

Comment:
This is too broad. I just have to add `2>&1` to my command to bypass any SP-related filtering :/